tests/drivers/flash/common/src/main.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2022 Nordic Semiconductor ASA
```

Comment:
Please follow [Commit Message Guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines)
`tests: drivers: flash: Modify driver to run with no-erase devices `